### DISCUSSION TITLE
ipkg-build: line 13: command: not found

### DISCUSSION THREAD

**Discussion by gray1717** [Entware/Entware#1085] [Q&A]
Не могу собрать ipk этим скриптом https://github.com/Entware/Entware/blob/master/scripts/ipkg-build
получаю ошибку ````ipkg-build: line 13: command: not found````, команда ````command```` на моем устройстве ````Linux RT-AC86U-C158 4.1.27 #2 SMP PREEMPT Sat Jul 20 13:50:04 EDT 2024 aarch64 ASUSWRT-Merlin````  отсутствует. 

Помогите, чего не хватает на моём роутере для успешного запуска этого скрипта, даже не знаю как правильно загуглить что есть ````command```` в каком пакете оно есть, знаю что тут его нет ````busybox --list````

**Comment by ryzhovau** [comment]
Это часть OpenWrt Buildroot, предназначенная для работы на определённом стенде сборки. 
https://openwrt.org/docs/guide-developer/toolchain/install-buildsystem
